config/api.yml

```diff
@@ -1120,10 +1128,6 @@
         :identifier: ems_infra_protect
       - :name: unassign
         :identifier: ems_infra_protect
-    :cloud_networks_subcollection_actions:
-      :get:
-      - :name: show
-      - :identifier: miq_cloud_networks_view
```

Comment:
same thinking, why moving them out of here ?  this gives us finer granularity on sub*/roles for the clusters collection